packages/components/src/color-palette/index.js

```diff
@@ -17,15 +17,17 @@ import Button from '../button';
 import Dropdown from '../dropdown';
 import Tooltip from '../tooltip';
 
-export default function ColorPalette( { colors, disableCustomColors = false, value, onChange, className } ) {
+export default function ColorPalette( { colors, disableCustomColors = false, value, onChange, className, filterColors } ) {
```

Comment:
To me filtering colors does not seem a responsibility of the ColorPalette component, why not pass the colors already filtered?